### PR TITLE
Add mutation for table rename

### PR DESCRIPTION
**Pull request by k9845**
Addresses
- https://github.com/the-dive/dive-server/issues/33

## Changes
- Add mutation for table rename

Mention related users here if any.

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] n+1 queries
- [x] flake8 issues
- [x] `print`
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [x] tests
- [ ] permission checks (tests here too)
- [ ] translations

### CODE REVIEW

File: apps/core/tests/test_mutations.py [hunk 1]
```diff
@@ -222,3 +222,30 @@ def test_update_table_properties(self):
...
+                    }
+                }
+            }
+        """
+        variables = {"tableId": table.id, "name": "New Table"}
+        resp_data = self.query_check(
+            mutate_query,
+            variables=variables,
+        )
+        print(resp_data)
```
Comment (bewakes) [inline apps/core/tests/test_mutations.py:248]: Missing print.

File: apps/core/mutations.py [hunk 2]
```diff
@@ -164,9 +164,37 @@ def mutate(root, info, id):
...
+        id = graphene.ID(required=True)
+        name = graphene.String(required=True)
+
+    errors = graphene.List(graphene.NonNull(CustomErrorType))
+    ok = graphene.Boolean()
+    result = graphene.Field(TableType)
+
+    @staticmethod
+    def mutate(root, info, id, name):
+        try:
```
Comment (bewakes) [inline apps/core/mutations.py:178]: This logic(`try` `except`) is repeated a lot of times. Maybe we should think about abstracting this out, perhaps via decorators. Not urgent, but something to think about.

Comment (k9845) [reply apps/core/mutations.py:178]: Sure

File: apps/core/tests/test_mutations.py (hunk 1 above)
Comment (k9845) [reply apps/core/tests/test_mutations.py]: Thanks

**Review by bewakes — APPROVED** [review]

**Pull request merged**